KubeArmor/common/common.go

```diff
@@ -28,6 +28,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// OuterKey struct
+type OuterKey struct {
+	PidNs uint32
+	MntNs uint32
+}
+
```

Comment:
Usage of Global Variable Map 🤔, not sure if there's any better way to handle due to cross module usage

// @DelusionalOptimist